conf/cmi/field.field.node.landing_page.field_content.yml

```diff
@@ -62,6 +62,7 @@ settings:
       service_list: service_list
       service_list_search: service_list_search
       unit_contact_card: unit_contact_card
+      unit_search: unit_search
```

Comment:
Add `sote_section` paragraph here so it can be used.

Reply:
These configs are now added.